src/wrapper/clean.rs

```diff
@@ -136,6 +136,23 @@ fn clean_dir_wrapper(path: &Path, dry_run: bool) -> Result<(), String> {
     Ok(())
 }
 
+fn clean_everything(wrapper: &Wrapper, global: bool, dry_run: bool)
+    -> Result<(), String>
+{
+    if global {
```

Comment:
In global case:

1. Don't remove the cache_dir and storage_dir itself, because we haven't created them (and we don't do that because we assume that they are often mounted volumes, so removing them would just lead to confusing error)
2. In case there is neither cache dir nor storage dir we must print some warning that `_clean --global` doesn't make sense.